sentinelhub/constants.py

```diff
@@ -343,6 +343,9 @@ def projection(self):
         :return: pyproj projection class
         :rtype: pyproj.Proj
         """
+        if pyproj.__version__ >= '2':
+            pyproj.Proj(self.epsg, preserve_units=True)
+
```

Comment:
Return missing.

Reply:
Nice catch. Unfortunately the problem goes deeper - it looks like this object cannot be initialized like this, otherwise it doesn't work correctly. I will investigate further.

Reply:
I decided to revert this back to the way it was. Further explanation is here: https://github.com/sentinel-hub/sentinelhub-py/issues/73